src/components/map/ha-map.ts

```diff
@@ -159,6 +159,7 @@ export class HaMap extends ReactiveElement {
     const map = this.shadowRoot!.getElementById("map");
     map!.classList.toggle("dark", darkMode);
     map!.classList.toggle("forced-dark", this.darkMode);
+    map!.classList.toggle("foced-light", !darkMode);
```

Comment:
`forced-light` should only be set when `this.darkMode === false` otherwise it will also add the forced class when then theme is actually light mode. `forced` here means the map dark mode is different than the dark mode of the theme.